internal/integration_test/fuzzcases/testdata/1802.wat

```diff
@@ -0,0 +1,29 @@
+(module
+  (func (;0;)
+    (local i32)
+      loop  ;; label = @4
+        loop ;;(result i32)  ;; label = @5
+          global.get 0
+          i32.eqz
+          if  ;; label = @6
+            unreachable
+          end
+          global.get 0
+          i32.const 1
+          i32.sub
+          global.set 0
+          block ;;(result i64)  ;; label = @6
+            i32.const 0
+            v128.load32_splat offset=32768 ;; <-- value in x11 is incorrect
```

Comment:
Could you delete this comment as it can easily be invalid and arm64 specific

Reply:
whoops, sure this was a note to self